src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java

```diff
@@ -2665,12 +2681,13 @@ private void connectHelper(ServerPortPlaceHolder serverInfo, int timeOutSliceInM
 
         // if the timeout is infinite slices are infinite too.
         tdsChannel = new TDSChannel(this);
+        InetSocketAddress inetSocketAddress;
         if (0 == timeOutFullInSeconds)
```

Comment:
I know it was like this originally, but we might as well add brackets to this if statement as well.